modules/addBattons.js

```diff
@@ -0,0 +1,55 @@
+import { usersSettings } from './usersSettings.js'
+
+// Добавляет кнопки пользовательского интерфейса
+function addBattons() {
+    let footer = document.createElement("div")
+    footer.className = "b-feedback"
+    let xc = sostavPerDay(usersSettings.usersSS)
+    footer.innerHTML = `${xc} <div id="activeStat" class="tampleteButton" style="height:23px">Статистика</div>`;
+
+    let classBlok = document.createElement("style")
+    classBlok.innerHTML = "div.tampleteButton{height: 30px;width: 130px;-webkit-appearance: button;-webkit-writing-mode: horizontal-tb !important;color: buttontext;text-shadow: none;display: inline-block;text-align: center;align-items: flex-start;background-color: buttonface;box-sizing: border-box;margin: 0em;font: 400 13.3333px Arial;padding: 1px 6px;border-width: 2px;border-style: outset;border-color: buttonface;border-image: initial;}}";
```

Comment:
you can separate html from css. Create a module css and [ut all the styling there ;)